### ISSUE TITLE
Issues with Fine-tuning

### ISSUE THREAD

**Issue by ilovecv** [johannakarras/DreamPose#25]
Hi,

I am fine-tuning the U-Net and VAE decoder with data from demo/sample/train. The motion seems correct, however, the generated images are kind of strange, below is one of the generated frames:
![050](https://user-images.githubusercontent.com/11083384/233745336-e76a778d-4993-4802-b688-f2cc63da0140.png)

This is the input image:
![key_frame](https://user-images.githubusercontent.com/11083384/233745390-46573835-4fa6-470c-b5bd-540310ecc2b4.png)

And this is the generated video using the trained checkpoints:

https://user-images.githubusercontent.com/11083384/233745437-cd277ebe-162f-4540-aa13-bbb1390d2847.mp4

Thank you very much for your help!

**Comment by h4rk8s** [comment]
same as I @johannakarras

**Comment by weizmann** [comment]
The same as I am @johannakarras 

![image](https://user-images.githubusercontent.com/2306111/233895831-10e787ce-70b3-41ba-9e52-420871aba936.png)

**Comment by zhangtao22** [comment]
I got similar results,the pics are more white and the movements are wired.But if I use author's pretrained model parameters. I can got author's results. What are the problems.

**Comment by johannakarras** [comment]
Thank you everyone for your interest in the project and your feedback! I have tested and re-uploaded all scripts for finetuning and uploaded the result to media/demo.gif. I hope this helps!

**Comment by weizmann** [comment]
I am wondering why those commits(add resource files / delete codes(the default value for `with_prior_preservation` is False)) would offer any help...

![sD1dnXE9vw](https://user-images.githubusercontent.com/2306111/236365423-08e5e0b0-063f-4313-b5f0-98486974a317.jpg)

**Comment by shenzy08** [comment]
I test the new scripts, but the problem still exists unfortunately.

**Comment by weizmann** [comment]
> 

Yeah, I think it is better to reopen the issue, and avoid more people from wasting time.

**Comment by weizmann** [comment]
> I test the new scripts, but the problem still exists unfortunately.

I think it is better to reopen the issue, and avoid more people from wasting time.

**Comment by johannakarras** [comment]
Okay, I was finally able to reproduce the issue. It is simply a problem with the normalization of the output image. It is a simple fix -- just add the following right before saving the final image inside of test.py: 

```
image = image - np.min(image)
image = (255*(image / np.max(image))).astype(np.uint8)
```

I will also update the repo.

Here is an example output frame after re-normalizing:
![pred_#51](https://user-images.githubusercontent.com/31581146/236571162-1b40fba9-c7f8-44bf-8204-7ade98ccae30.png)

**Comment by zhangtao22** [comment]
@johannakarras I think the problem doesn't lie in here

**Comment by weizmann** [comment]
> The same as I am @johannakarras
> 
> Character instance is not stabilized between consecutive frames...
> 
> ![image](https://user-images.githubusercontent.com/2306111/233898256-104eefdb-8629-470a-9ef5-aedcc6b362c8.png)

@johannakarras The head size varies drastically between consecutive frames.
